tests/src/test/scala/cats/tests/FoldableSuite.scala

```diff
@@ -218,14 +218,6 @@ class FoldableSuiteAdditional extends CatsSuite {
     checkFoldMStackSafety[Vector](_.toVector)
   }
 
-  test("Foldable[Set].foldM stack safety") {
-    checkFoldMStackSafety[Set](_.toSet)
-  }
```

Comment:
shall we replace these two foldMStackSafety tests with SortedSet and SortedMap ones?